addon/components/paper-switch.js

```diff
@@ -47,21 +70,6 @@ export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
     let switchHammer = new Hammer(element);
     this.switchHammer = switchHammer;
```

Comment:
let's prefix hammer variables with `_` e.g `this._switchHammer` and `this._thumbElementHammer`.